draft-birkholz-scitt-software-use-cases.md

```diff
@@ -167,6 +167,35 @@ There is no standardized way to:
 * to check for the trustworthiness of a provider
 * to check the integrity of modifications or transformations done by a provider
 
+## Post-Boot Firmware Provenance
+
+In contrast to operating systems or user space software components of a large and complex systems, firmware components are often already executed during boot-cycles before there is an opportunity to authenticate them.
+
+Authentication takes place, for example, by validating a signed artefact against a Reference Integrity Manifest (RIM).
+Corresponding procedures are often called authenticated, measured, or secure boot. The output of these high assurance boot procedures is often used as input to more complex verification known as remote attestation procedures.
+
+If measurements before execution are not possible, static after-the-fact analysis is required, typically by examining artifacts.
```

Comment:
Ok, just noting we're not consistent in the same file :)

> ... by examining artifacts.

Reply:
see above :-)